python/lsst/fgcmcal/fgcmBuildStars.py

```diff
@@ -600,6 +619,13 @@ def fgcmMakeAllStarObservations(self, groupedDataRefs, visitCat):
         instMagKey = outputSchema['instMag'].asKey()
         instMagErrKey = outputSchema['instMagErr'].asKey()
 
+        # Prepare local background if desired
+        if self.config.doSubtractLocalBackground:
+            localBackgroundFluxKey = sourceSchema[self.config.localBackgroundFluxField].asKey()
+            localBackgroundArea = np.pi * self.config.calibFluxApertureRadius**2.
```

Comment:
Gentle reminder (here and throughout) of https://developer.lsst.io/python/style.html#binary-operators-should-be-surrounded-by-a-single-space-except-for